Dockerfile.dapper

```diff
@@ -20,7 +20,13 @@ ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 \
     DOCKER_URL_arm64=https://github.com/rancher/docker/releases/download/v1.10.3-ros1/docker-1.10.3_arm64 \
     DOCKER_URL=DOCKER_URL_${ARCH}
 
-RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
+RUN if [ "${ARCH}" == "s390x" ]; then \
+        curl -L https://download.docker.com/linux/static/stable/s390x/docker-18.06.3-ce.tgz | tar xzvf - && \
+        cp docker/docker /usr/bin/ && \
+        chmod +x /usr/bin/docker; \
+    else \
+        wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker; \
+    fi
```

Comment:
Can this be changed to use the same pattern as above?

Reply:
Unfortunately no, docker binary for s390x is inside a compressed file and for s390x, tar is necessary.

We can get everything into the same pattern, if we can update the docker from 1.10.3 to 18.06.3. But the problem here is for arm, docker binary is being used from a fork repo of rancher https://github.com/rancher/machine/blob/master/Dockerfile.dapper#L19. So not sure if we can use docker binary from https://download.docker.com/linux/static/stable/.